module/Activity/src/Activity/Controller/ActivityController.php

```diff
@@ -61,12 +66,17 @@ public function createAction()
             $form->setData($this->getRequest()->getPost());
 
             if ($form->isValid()) {
-                $activity = $activityService->createActivity($form->getData());
-
+                //var_dump($form->getData(\Zend\Form\FormInterface::VALUES_AS_ARRAY));//debug
+                $activity = $activityService->createActivity($form->getData(\Zend\Form\FormInterface::VALUES_AS_ARRAY));
+                
                 $this->redirect()->toRoute('activity/view', array(
                     'id' => $activity->get('id'),
                 ));
             }
+            else {
+                echo 'Form is invalid!';
+                echo var_dump($form->getInputFilter()->getMessages());
+            }
```

Comment:
Ja, dit is toch iets meer WIP dan je misschien verwacht had. Hetzelfde geldt voor de volgende 2 opmerkingen.